frontend/src/main/components/Users/UsersTable.js

```diff
@@ -23,7 +23,17 @@ const columns = [
         id: 'admin',
         accessor: (row, _rowIndex) => String(row.admin) // hack needed for boolean values to show up
     },
+    {
```

Comment:
I suggest that you do not try to combine this change in the same PR as changing the entity to also include the driver.  It isn't necessary, and it complicates things.